backend-API-routes/backend-api-routes_products/pages/products/index.js

```diff
@@ -0,0 +1,24 @@
+import useSWR from "swr";
+
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+export default function Products() {
+const { data } = useSWR(`/api/products`, fetcher);
+console.log("data", data);
+if (!data) {
+    return <div>Loading.....</div>
+}
+
+return (
+    <>
+    <ul>
+        {data.map((product) => (
+            <li key={product.data}>
```

Comment:
Not sure what exactly is this product.data, on my I used product.id. But if it works, that's it.